src/main/lombok/no/hvl/dat250/h2020/group5/entities/Poll.java

```diff
@@ -68,6 +68,11 @@ public boolean addVote(Vote vote) {
     return true;
   }
 
+  public void addOwner(User user) {
+    this.setPollOwner(user);
+    user.getUserPolls().add(this);
+  }
+
```

Comment:
Veldig bra! 👍 

Som beskrevet i #88 kan det bli litt forvirrende med `addOwner(owner)` og `setOwner(owner)` så hva med: 

```suggestion
  public void setOwnerAndAddThisPollToOwner(User user) {
    this.owner = user;
    user.getUserPolls().add(this);
  }
```
I User:
```suggestion
  public void addPollAndSetThisUserAsOwnerInPoll(Poll poll) {
    userPolls.add(poll);
    poll.setOwner(this);
  }

```